src/utils/document_utils.py

```
@@ -300,9 +303,9 @@ async def update_source_info(client: QdrantClient, source_id: str, summary: str,
             ],
             wait=True
         )
-        print(f"Upserted source: {source_id}")
+        logger.info(f"Upserted source: {source_id}")
```

Ruff (0.12.2)

Logging statement uses f-string

(G004)